send/generic_send

```diff
@@ -84,6 +105,16 @@ case $DESTINATION_TYPE in
 		HOSTNAME="$DESTINATION"
 		HOST="$DESTINATION"
 		;;
+	${DESTINATION_TYPE_USER_HOST_WINDOWS})
+		HOST=`echo $DESTINATION | sed -e 's/:.*//'`
+		HOSTNAME=`echo $HOST | sed -e 's/^.*@//'`
+		FILTER="base64"
+		;;
+	${DESTINATION_TYPE_HOST_WINDOWS_PROXY})
+		if [ -z "${WINDOWS_PROXY}" ]; then echo 'Windows proxy name is not defined in /etc/perun/services/generic_send/generic_send.conf' >&2 ; exit 1; fi
```

Comment:
Put the variable name to the error message. It will be easier to debug.

Also, I would mentioned that the variable is **usually** defined in /etc/perun/...

Reply:
Fixed